docs/setup/jetson.md

```diff
@@ -118,7 +118,7 @@ The recommended way to run ROS2 on Jetson is to use pre-built Docker container i
 
 1. Set the default network interface by setting Cyclone DDS configuration.
 
-        export CYCLONEDDS_URI='<CycloneDDS><Domain><General><NetworkInterfaceAddress>l4tbr0</NetworkInterfaceAddress></General></Domain></CycloneDDS>'
+        export CYCLONEDDS_URI='<CycloneDDS><Domain><General><Interfaces><NetworkInterface name="l4tbr0" /></Interfaces></General></Domain></CycloneDDS>'
```

Comment:
Are the previous instructions outdated or not the correct way?  If the former, does this updated config work for our older releases?

Reply:
Ah, I should have mentioned. The old style (`NetworkInterfaceAddress`) does still work, but the new style is available and recommended for ROS Humble and future releases.

Reply:
No, as far as I can see Galactic is on the CycloneDDS 0.8.x release train so there the new style is not implemented.

Reply:
Should we break out separate instructions for galactic and humble where the configs differ between the two

Reply:
That is probably a good idea then. Do you want (1) two separate files for humble/galactic or (2) mention the difference inline? Also, do you want me to change my contribution or restructure it yourself? I'm okay with either :slightly_smiling_face:

Reply:
@alsora What is your opinion?

Reply:
I think that for now we should go with a disclaimer / inline comment.
Eventually we will fully support multiple versions for the docs and we can take care of it appropriately